### PR TITLE
Update walkthrough doc to implement guestbook-operator

### PR DESCRIPTION
**Pull request by atoato88**
This PR updates `docs/addon/walkthrough/README.md` contents to implement guestbook-operator, instead dashboard-operator.

related to: #80

### CODE REVIEW

File: docs/addon/walkthrough/README.md [hunk 1]
```diff
@@ -520,19 +532,19 @@ status that can be surfaced in various user interfaces.
 	curl https://raw.githubusercontent.com/kubernetes-sigs/application/master/config/crds/app_v1beta1_application.yaml -o config/crd/app_v1beta1_application.yaml
 	```
 
-1. Add an instance of the Application CR in your manifest:
+2. Add an instance of the Application CR in your manifest:
```
Comment (atoato88) [inline docs/addon/walkthrough/README.md:535]: I updated this section for guestbook-oprator, but current guestbook-operator doesn't include Aplication CR.
Which is better to implement Application CR on guestbook-operator or omit this section?
Sorry for that I don't know details about Application CR...

**Comment by atoato88** [comment]
/retitle Update walkthrough doc to implement guestbook-operator

File: docs/addon/walkthrough/README.md (hunk 1 above)
Comment (johnsonj) [reply docs/addon/walkthrough/README.md:534]: This section should walk through configuring the guestbook operator to manage an Application CR.  The two reconcile options added (`declarative.WithManagedApplication(r.watchLabels), declarative.WithObjectTransform(addon.TransformApplicationFromStatus)`) handle this. These updates look good to me.

The Application CR is meant to bubble up high level information/groupings of a set of resources in the cluster. It's managed by sig-apps.

**Comment by johnsonj** [comment]
/lgtm
/approve

thank you @atoato88 ! If you have more open questions on the guestbook-operator/application CR please feel free to follow up

**Pull request merged**